components/text_sensor/modbus_controller.rst

```diff
@@ -18,11 +18,10 @@ Configuration variables:
     - holding: Holding Registers - Holding registers are the most universal 16-bit register. Read and Write access
     - read: Read Input Registers - registers are 16-bit registers used for input, and may only be read
 - **address**: (**Required**, int): start address of the first register in a range
-- **bitmask**: (*Optional*) some values are packed in a response. The bitmask can be used to extract a value from the response.  For example, if the high byte value register 0x9013 contains the minute value of the current time. To only exctract this value use bitmask: 0xFF00.  The result will be automatically right shifted by the number of 0 before the first 1 in the bitmask.  For 0xFF00 (0b1111111100000000) the result is shifted 8 posistions.  More than one sensor can use the same address/offset if the bitmask is different.
```

Comment:
did you mean to remove this? It's used in the example too and is still around.

Reply:
I removed bitmask because it is not used in the code (https://github.com/esphome/esphome/blob/dev/esphome/components/modbus_controller/text_sensor/modbus_textsensor.cpp). I think it doesn't even make sense for a text sensor.